src/pages/documentation/consent-for-edge-network/api-reference.md

```diff
@@ -64,3 +64,5 @@ Android
 iOS
 
 <Tabs query="platform=ios&api=update-consents"/>
+
+For additional information about the management of Consent for Edge Network, please refer to [Privacy and GDPR](../resources/privacy-and-gdpr.md#using-experience-platform-sdks-for-edge-network).
```

Comment:
... the management of consent preferences, please refer ...

Reply:
"... please refer to the [Privacy and GDPR documentation].